mmv1/products/dns/api.yaml

```diff
@@ -493,3 +493,65 @@ objects:
         description: |
           As defined in RFC 1035 (section 5) and RFC 1034 (section 3.6.1)
         api_name: rrdatas
+  - !ruby/object:Api::Resource
+    name: 'ResourceDnsRecordSet'
+    kind: 'dns#resourceRecordSet'
+    min_version: beta
```

Comment:
Last time I reviewed this we weren't removing the old resource right? If we do that while having `min_version: beta` we'll be removing the entire resource from the GA provider

Reply:
Sorry on the iterations here, but we should have both resources or make this GA

Reply:
@c2thorn now the resource is supported by both GA/Beta versions. I made the change to accommodate that, please have a look